### PR TITLE
Add user response time and extensions configs

### PR DESCRIPTION
**Pull request by mihirukongahage**
### Proposed changes in this pull request

 - Add user response time and extensions configs

Resolves: [#9714](https://github.com/wso2/product-is/issues/9714)

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 1]
```diff
@@ -2499,6 +2499,9 @@
                 <Origin>{{origin}}</Origin>
             {% endfor %}
         </FIDO2TrustedOrigins>
+        <!-- Time allocated for the users to authenticate -->
+        <UserResponseTimeout>{{fido.user_response_timeout}}</UserResponseTimeout>
```
Comment (Pushpalanka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2503]: What is the time unit? (eg. mins, seconds). There is a way do define time unit in J2 as I have heard. If not indicate in the comment.

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 2]
```diff
@@ -2499,6 +2499,9 @@
...
+        <Extensions>{{fido.extensions}}</Extensions>
```
Comment (Pushpalanka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2504]: If it's extensions, shouldn't it be like
```
<Extensions>
    <Extension></Extension>
    <Extension></Extension>
</Extensions>
```
Does this syntax support it?

**Comment by somindatommy** [comment]
@mihirukongahage Can you accept the `Contributor License Agreement`?

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 (hunk 1 above)
Comment (mihirukongahage) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2507]: Addressed with https://github.com/wso2/carbon-identity-framework/pull/3360/commits/c1450738df06e6901287350c2ce5d69962cc02df

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 3]
```diff
@@ -2499,6 +2499,13 @@
                 <Origin>{{origin}}</Origin>
             {% endfor %}
         </FIDO2TrustedOrigins>
+
+        <!--
+            Time allocated for the users to authenticate
+            Time configurations are in milliseconds
```
Comment (sachiniWettasinghe) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2505]: Fix formatting. Both comments should end with a full stop.

Comment (mihirukongahage) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Addressed with https://github.com/wso2/carbon-identity-framework/pull/3360/commits/debac20f6574d2b76a897bb1794f84d065703ed1

**Review by sachiniWettasinghe — APPROVED** [review]

**Review by Pushpalanka — APPROVED** [review]

**Pull request merged**